src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -79,6 +84,13 @@
     @Argument(fullName = StandardArgumentDefinitions.ADD_OUTPUT_SAM_PROGRAM_RECORD, shortName = StandardArgumentDefinitions.ADD_OUTPUT_SAM_PROGRAM_RECORD, doc = "If true, adds a PG tag to created SAM/BAM/CRAM files.", optional=true)
     public boolean addOutputSAMProgramRecord = true;
 
+    // default value of 40MB based on a test with CountReads (it's 5x faster than no prefetching)
+    @Argument(fullName = StandardArgumentDefinitions.CLOUD_PREFETCH_BUFFER_LONG_NAME, shortName = StandardArgumentDefinitions.CLOUD_PREFETCH_BUFFER_SHORT_NAME, doc = "Size of the cloud-only prefetch buffer (in MB; 0 to disable).", optional=true)
+    public int cloudPrefetchBuffer = 40;
+
+    @Argument(fullName = StandardArgumentDefinitions.CLOUD_INDEX_PREFETCH_BUFFER_LONG_NAME, shortName = StandardArgumentDefinitions.CLOUD_INDEX_PREFETCH_BUFFER_SHORT_NAME, doc = "Size of the cloud-only prefetch buffer (in MB; 0 to disable).", optional=true)
+    public int cloudIndexPrefetchBuffer = 5;
```

Comment:
I haven't run tests yet, so this value is just a guess based on the fact that the indices I have are small.

You make a good point that having a single argument is easier. So let's allow the user to either only set cloudPrefetchBuffer (in which case this value is used for both the data and index), or to set both (if they want to optimize further; after all it would be a great concidence if the best value were the same for both data types). As a side effect this currently makes the default value be the same for both (we can change this later if needed).